test/utils/cTokenAddresses.ts

```diff
@@ -0,0 +1,42 @@
+import { keccak256 } from "./keccak256";
+const parameters = require("../../configuration/parameters");
+
+type ParameterConfig = [
+  string,
+  number,
+  {
+    // "NAME": "ETH",
+    cToken: string;
+    underlying: string;
+    symbolHash: string;
+    baseUnit: string;
+    priceSource: string;
+    fixedPrice: string;
+    uniswapMarket: string;
+    reporter: string;
+    reporterMultiplier: string;
+    isUniswapReversed: boolean;
+  }[]
+];
+
+interface TokenAddresses {
+  [T: string]: {
+    underlying: string;
+    cToken: string;
+  };
+}
+
+export const getTokenAddresses = (symbols: string[]): TokenAddresses => {
+  return symbols.reduce((prev, curr) => {
+    const symbolHash = curr === "WBTC" ? keccak256("BTC") : keccak256(curr);
```

Comment:
Is WBTC the only special case? What about ETH/WETH?

Reply:
We don't support `WETH` according to the `parameters.js` file.

Reply:
Hey guys are we covering all the tokens in the parameters.js file? @yosriady @cds95